includes/Classifai/Providers/OpenAI/ChatGPT.php

```diff
@@ -717,6 +814,109 @@ public function generate_titles( int $post_id = 0, array $args = [] ) {
 		return $return;
 	}
 
+	/**
+	 * Resizes content.
+	 *
+	 * @param int   $post_id The Post Id we're processing
+	 * @param array $args Arguments passed in.
+	 * @return string|WP_Error
+	 */
+	public function resize_content( int $post_id, array $args = array() ) {
+		if ( ! $post_id || ! get_post( $post_id ) ) {
+			return new WP_Error( 'post_id_required', esc_html__( 'Post ID is required to resize content.', 'classifai' ) );
+		}
+
+		$settings = $this->get_settings();
+		$args     = wp_parse_args(
+			array_filter( $args ),
+			[
+				'num' => $settings['number_resize_content'] ?? 1,
+			]
+		);
+
+		$request = new APIRequest( $settings['api_key'] ?? '' );
+
+		if ( 'shrink' === $args['resize_type'] ) {
+			$prompt = 'Decrease the content length no more than 2 to 4 sentences.';
+		} else {
+			$prompt = 'Increase the content length no more than 2 to 4 sentences.';
+		}
+
+		/**
+		 * Filter the prompt we will send to ChatGPT.
+		 *
+		 * @since 2.3.0
+		 *
+		 * @param {string} $prompt Prompt we are sending to ChatGPT. Gets added before post content.
+		 * @param {int} $post_id ID of post we are summarizing.
+		 * @param {array} $args Arguments passed to endpoint.
+		 *
+		 * @return {string} Prompt.
+		 */
+		$prompt = apply_filters( 'classifai_chatgpt_' . $args['resize_type'] . '_content_prompt', $prompt, $post_id, $args );
+
+		/**
+		 * Filter the request body before sending to ChatGPT.
+		 *
+		 * @since 2.3.0
+		 * @hook classifai_chatgpt_resize_content_request_body
+		 *
+		 * @param {array} $body Request body that will be sent to ChatGPT.
+		 * @param {int}   $post_id ID of post we are summarizing.
+		 *
+		 * @return {array} Request body.
+		 */
+		$body = apply_filters(
+			'classifai_chatgpt_resize_content_request_body',
+			[
+				'model'       => $this->chatgpt_model,
+				'messages'    => [
+					[
+						'role'    => 'system',
+						'content' => $prompt,
+					],
+					[
+						'role'    => 'user',
+						'content' => esc_html( $args['content'] ) . '',
```

Comment:
Do we need the extra concatenation here?
```suggestion
						'content' => esc_html( $args['content'] ),
```

Reply:
@Sidsector9 Just following up again on this one